### PR TITLE
Simple client server

### PR DESCRIPTION
**Pull request by atifzia**

### CODE REVIEW

File: ClientServer.go [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+import (
+	"bufio"
+	"fmt"
+	"net"
+	"os"
+	"strings"
+	"sync"
+)
+
+var wg sync.WaitGroup
```
Comment (furusiyya) [inline ClientServer.go:12]: avoid global variables

File: ClientServer.go [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+		if message == "exit\n" {
+			conn.Close()
+			break
+		}
+
+		// process the recieved string from client
+		newmessage := strings.ToUpper(message)
+		// send new string back to client
+		conn.Write([]byte(newmessage + "\n"))
```
Comment (furusiyya) [inline ClientServer.go]: Write() does not return error?

Comment (atifzia) [reply ClientServer.go]: yes, it returns

File: ClientServer.go (hunk 1 above)
Comment (atifzia) [reply ClientServer.go:12]: got it

**Pull request merged**